oeps/processes/oep-0060-proc-sec-group.rst

```diff
@@ -0,0 +1,243 @@
+
+The Security Working Group will provide this numerical score and qualitative severity rating when notifying maintainers. This is so maintainers are better informed on how severe the vulnerability is and have a sense of how important it may be to interrupt their work on the repository to resolve the vulnerability.
+
+.. _Common Vulnerability Scoring System (CVSS), version 3.1: https://www.first.org/cvss/v3.1/specification-document
+
+Maintainer Responsibility
+-------------------------
+Maintainers are ultimately responsible for resolving disclosed security vulnerabilities.
+
+To assist the maintainer with tracking their repository's vulnerabilities, the Security Working Group will create a `GitHub repository security advisory`_ pre-assigned with the scored severity.
+
+Maintainers should inform the Security Working Group if they judge a vulnerability to be a different severity than what was originally triaged. The Security Working Group must accept the maintainer's adjudication, but should comment on any considerations around the adjudication.
+
+Maintainers will be routinely reminded to remediate disclosures in a frequency proportional to the severity of the disclosure. The following table shows the default reminder frequency until resolution of each severity classification:
+
+========    =====   ==================
+Severity    Score   Reminder frequency
+========    =====   ==================
+Low         ≥0.1    Twice a year
+Medium      ≥4.0    Once a quarter
+High        ≥7.0    Once a month
+Critical    ≥9.0    Once a week
+========    =====   ==================
+
+The on-call Security Working Group member is responsible for sending reminders for any vulnerability reminders due during the time they are on-call. These routine reminders should be batched so that all reminders of the same frequency for a maintainer are compiled into a single notification.
+
+.. note::
+
+    Maintainers are encouraged to reach out to security@openedx.org if they need any help remediating security issues.
+
+.. _GitHub repository security advisory: https://docs.github.com/en/code-security/security-advisories/repository-security-advisories/creating-a-repository-security-advisory
+
+Security Vulnerability Privacy
+------------------------------
+Details of security vulnerabilities will remain private until a fix is released. Members of the Open edX community with access to the details of a disclosure are asked to keep those details private until a fix is publicly released for the vulnerability.
+
+Maintainers and their delegates should use GitHub's `temporary private fork`_ feature within the GitHub repository security advisory created by the Security Working Group to keep the implementation details of a fix for a vulnerability private until the appropriate time to release the fix to the public.
```

Comment:
1. Is this "should" or "must"? 
2. Can a maintainer decide to use a public PR instead, and pretend that the fix isn't public until merged? Does it depend on severity? Would there be repercussions for being able to stay a maintainer?
3. If someone mistakenly creates a public fix PR, should we lay out the process of how to handle? Is it simply to close the PR and delete the branch? Should the PR subject and description be cleaned as-needed? Other?

Reply:
Updated language to better reflect expectations around security vulnerability privacy. 

As far as your second question, if a maintainer purposefully chooses to make a patch public, the process for deciding what to do would better fall under the Maintainer OEP, specifically [Removing a Maintainer](https://github.com/openedx/open-edx-proposals/blob/f43defa5cb60ac0534921bf5d4427e515dfdf0f5/oeps/processes/oep-0055-proc-project-maintainers.rst#removing-a-maintainer) if it has to go that far.